examples/pinball/run-transient.py

```diff
@@ -0,0 +1,40 @@
+import psutil
+
+import hydrogym.firedrake as hgym
+
+output_dir = "."
+pvd_out = None
+restart = None
+checkpoint = "checkpoint-coarse.h5"
+
+flow = hgym.Pinball(Re=10, h5_file=restart, mesh="coarse")
```

Comment:
Are we still using the `hf_file` kwarg? I thought I had seen that checkpoints are loaded with the `restart` keyword. https://github.com/dynamicslab/hydrogym/blob/main/hydrogym/core.py#L54

Reply:
Thanks, good catch.  I copy-pasted from the demo script, which is also out of date.